From 5cf409eea959b69461f6b41e545714373aa436e9 Mon Sep 17 00:00:00 2001
From: Jonathan Nieder <jrnieder@gmail.com>
Date: Thu, 6 Jan 2011 22:43:05 -0600
Subject: [PATCH 4/6] pre-rebase hook: capture documentation in a <<here
 document

With this change, the sample hook passes a syntax check (sh -n).

Signed-off-by: Jonathan Nieder <jrnieder@gmail.com>
---
 templates/hooks--pre-rebase.sample |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/templates/hooks--pre-rebase.sample b/templates/hooks--pre-rebase.sample
index 053f111..7fccd3d 100755
--- a/templates/hooks--pre-rebase.sample
+++ b/templates/hooks--pre-rebase.sample
@@ -90,6 +90,7 @@ fi
 
 exit 0
 
+<<\DOC_END
 ################################################################
 
 This sample hook safeguards topic branches that have been
@@ -167,3 +168,5 @@ To compute (2):
 	git rev-list master..topic
 
 	if this is empty, it is fully merged to "master".
+
+DOC_END
-- 
1.7.5.3

